.circleci/config.yml

```diff
@@ -167,6 +167,77 @@ jobs:
             terraform destroy -var project=${CLOUDSDK_CORE_PROJECT} -auto-approve
           when: always
 
+  acceptance-aks-1-19:
+    environment:
+      - TEST_RESULTS: /tmp/test-results
+    docker:
+      # This image is build from test/docker/Test.dockerfile
+      - image: docker.mirror.hashicorp.services/hashicorpdev/consul-helm-test:0.5.0
+
+    steps:
+      - checkout
+
+      - run:
+          name: terraform init & apply
+          working_directory: test/terraform/aks
+          command: |
+            terraform init
+
+            terraform apply -var client_id="$ARM_CLIENT_ID" -var client_secret="$ARM_CLIENT_SECRET" -var cluster_count=2 -auto-approve
```

Comment:
Where are we setting $ARM_CLIENT_ID and $ARM_CLIENT_SECRET?

Reply:
You can provide env variables to all jobs in circleci by setting them in [project settings](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project). If you go to our project settings for consul-helm, you should be able to see those.